config/deploy.js

```diff
@@ -101,6 +101,12 @@ function deployBuild(url, folder) {
         let next_path = path.resolve(folder);
         shell.cd(next_path);
         const start = Date.now();
+        if (url === EDITOR_URL) {
```

Comment:
Add a comment describing what you are doing here for future developers making changes to this file.